portfolio/src/main/java/com/google/sps/servlets/InteractiveChartServlet.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** TODO(briafassler): store data to a database */
+@WebServlet("/interactive-chart")
+public class InteractiveChartServlet extends HttpServlet {
+  private Map<String, Integer> emotionVotes = new HashMap<>();
```

Comment:
Nice use of a hash map here! Likely won't be used once you move over to a database though ):